### PR TITLE
✨ 💫  Support Multiple languages

### PR DESCRIPTION
**Pull request by nipunsadvilkar**
Languages to be supported:

- [x] English
- [x] Bulgarian
- [x] Spanish
- [x] Russian
- [x] Arabic
- [x] Amharic
- [x] Marathi
- [x] Hindi
- [x] Armenian
- [x] Persian
- [x] Urdu
- [x] Polish
- [x] Chinese
- [x] Dutch
- [x] Danish
- [x] French
- [x] Italian
- [x] Greek
- [x] Burmese
- [x] Japanese
- [x] Deutsch
- [x] Kazakh

### CODE REVIEW

File: tests/test_segmenter.py [hunk 1]
```diff
@@ -46,6 +46,16 @@ def test_exception_with_both_clean_and_span_true():
     assert str(e.value) == "char_span must be False if clean is True. "\
                             "Since `clean=True` will modify original text."
 
+def test_exception_with_otherthan_en_lang_span_true():
```
Comment (DeNeutoy) [inline tests/test_segmenter.py:49]: Is it possible to support the `char_span` functionality for any language with a corresponding spacy model?

**Comment by nipunsadvilkar** [comment]
Yes it would. Had pushed those changes since I was working on multiple languages. Also, I did some experiment with another approach to provide char_span functionality to other languages. Checkout branch `npn-char-span-multiple-lang` to see the approach - I've used custom tokenizer which tokenizes into characters and I set `is_sent_start` to True as per mapping from pysbd. Let me know your thoughts

**Comment by DeNeutoy** [comment]
@nipunsadvilkar I don't think that is the right approach - it needs to support char spans without modifying the tokenizer, as that would be quite unexpected behaviour and break other spacy components.

Why does the method that you already have here: https://github.com/nipunsadvilkar/pySBD/blob/master/pysbd/processor.py#L50

not work for other languages?

**Comment by nipunsadvilkar** [comment]
Haven't tried with other languages. Could work with english-like languages

File: tests/test_segmenter.py (hunk 1 above)
Comment (nipunsadvilkar) [reply tests/test_segmenter.py]: Removed dependency on spacy in this [recent commit](https://github.com/nipunsadvilkar/pySBD/pull/67/commits/7d75901742ec5b82f71558f39e8243da90976582)

File: tests/lang/test_amharic.py [hunk 2]
```diff
@@ -10,4 +10,5 @@
 def test_am_sbd(am_default_fixture, text, expected_sents):
     """Amharic language SBD tests"""
     segments = am_default_fixture.segment(text)
+    segments = [s.strip() for s in segments]
```
Comment (DeNeutoy) [inline tests/lang/test_amharic.py:13]: Why do the sentences now possibly have extra whitespace?

Comment (nipunsadvilkar) [reply tests/lang/test_amharic.py:13]: That's to make it nondestructive.

Example:
```python
import pysbd
text = "My name is Jonas E. Smith. Please turn to page no. 55."
seg = pysbd.Segmenter(language="en", clean=False, char_span=False)
segments = seg.segment(text)
# ['My name is Jonas E. Smith. ', 'Please turn to page no. 55.']
"".join(segments) 
# "My name is Jonas E. Smith. Please turn to page no. 55."
```
Earlier it was without trailing spaces. This is in a way important but anyway with spacy start_char index is imporant
`['My name is Jonas E. Smith.', 'Please turn to page no. 55.']`

Comment (DeNeutoy) [reply tests/lang/test_amharic.py:13]: Oh, perfect 👌

**Pull request merged**